src/Renderer.ts

```diff
@@ -828,7 +824,10 @@ export default class Renderer {
 
   public stamp(spr: Sprite | Stage): void {
     this._setFramebuffer(this._penSkin._framebufferInfo);
-    this._renderLayers(new Set([spr]), { renderSpeechBubbles: false });
+    this.renderSprite(spr, {
+      drawMode: ShaderManager.DrawModes.DEFAULT,
+      renderSpeechBubbles: false,
+    });
```

Comment:
This looks like a nicer interface for doing what we're doing anyway! Is there a reason you're passing `drawMode`, though? It's [provided as default](https://github.com/leopard-js/leopard/blob/514aa350bd546fed88b3c95c988e9afd8faff3a9/src/Renderer.ts#L293-L296) and just this would read nicer and be a more direct change:

```diff
-    this._renderLayers(new Set([spr]), { renderSpeechBubbles: false });
+    this.renderSprite(spr, { renderSpeechBubbles: false });
```

Reply:
Good call; that cleans up a lot of code